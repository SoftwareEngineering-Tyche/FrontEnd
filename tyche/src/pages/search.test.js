import { mount } from 'enzyme';
import Search from './search-page';

describe('renders <Search />', () => {
  it('should pass mount snapshot test', () => {
    const wrapper = mount(<Search />);
    expect(wrapper).toMatchSnapshot();
  });
});