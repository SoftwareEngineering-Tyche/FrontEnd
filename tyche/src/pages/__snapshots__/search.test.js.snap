// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders <Search /> should pass mount snapshot test 1`] = `ReactWrapper {}`;
